start

```diff
@@ -207,7 +207,16 @@ if [ "$1" = "runserver" ] ; then
         fi
         ln -s "/etc/supervisor/conf.d/$WEBLATE_SERVICE.conf" "$SUPERVISOR_CONF"
     else
-        ln -s /etc/supervisor/conf.d/*.conf "$SUPERVISOR_CONF"
+        # Symlink all non-celery services
+        find /etc/supervisor/conf.d -type f ! -name 'celery-*.conf' -print0 | xargs -0 -I '{}' ln -s '{}' "$SUPERVISOR_CONF"
+
+        if [ "${CELERY_SINGLE_PROCESS:-0}" -eq 1 ] ; then
+            # Symlink single process service only
+            ln -s /etc/supervisor/conf.d/celery-signle.conf  "$SUPERVISOR_CONF"
```

Comment:
Thank you for fixing up my PR! It is already merged, but I noticed a typo here. Strange that the tests didn't catch it?

Reply:
Thanks for spotting! Not sure why it didn't fail the tests, probably they don't verify that the celery service is running (whole container starts and web service runs).